src/flint/fmpz_mat.jl

```diff
@@ -907,30 +911,32 @@ end
 @doc raw"""
     lll(x::ZZMatrix, ctx::lll_ctx = lll_ctx(0.99, 0.51))
 
-Return the LLL reduction of the matrix $x$. By default the matrix $x$ is a
-$\mathbb{Z}$-basis and the Gram matrix is maintained throughout in
-approximate form. The LLL is performed with reduction parameters
-$\delta = 0.99$ and $\eta = 0.51$. All of these defaults can be overridden by
-specifying an optional context object.
+Return a matrix $L$ whose rows form an LLL-reduced basis of the
+$\mathbb{Z}$-lattice generated by the rows of $x$. $L$ may contain additional
+zero rows.
+
+By default, the Gram matrix is maintained throughout in approximate form and
+the LLL is performed with reduction parameters $\delta = 0.99$ and $\eta = 0.51$.
+All of these defaults can be overridden by specifying an optional context object.
```

Comment:
To (mis-)quote @fieker: talking about Gram matrix here does not quite make sense, as an `lll_ctx` is used with `rep=:zbasis`, so no Gram matrix involved...

Reply:
I know that Johannes did not add the sentence.

I do not see how the sentence about the "approximate form" is relevant. It is too underspecified to be of any use. I suggest we just delete it.

Reply:
So maybe
```suggestion
By default, the reduction parameters $\delta = 0.99$ and $\eta = 0.51$ are used.
These and other defaults can be overridden by specifying an optional context object.
```

Reply:
I just copied this part from the current documentation, I'm happy to delete it.